spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -88,3 +88,10 @@ See the xref:kafka/receiving-messages/message-listener-container.adoc#message-li
 
 It is now possible to get an observation for each record when using a batch listener.
 See xref:kafka/micrometer.adoc#batch-listener-obs[Observability for Batch Listeners] for more information.
+
+[[x40-kafka-queues]]
+=== Kafka Queues (Share Consumer) Support
+
+Spring for Apache Kafka now provides early access support for Kafka Queues through share consumers, which are part of Apache Kafka 4.0.0 and implement KIP-932.
+This enables cooperative consumption where multiple consumers can consume from the same partitions simultaneously, providing better load distribution compared to traditional consumer groups.
+See xref:kafka/kafka-queues.adoc[Kafka Queues (Share Consumer)] for more information.
```

Comment:
Just nip-pick for convenience.
If you leave brackets empty, the link text would be taken from the title of the pointing paragraph.
So, in our case that would still be same `Kafka Queues (Share Consumer)` text. 😄